### ISSUE TITLE
Lock-icon tooltip incorrectly says that all temporary scope changes are saved

### ISSUE THREAD

**Issue by adrienbeau** [gorhill/uMatrix#311]
Above the matrix, the lock icon has a tooltip that says "Save all temporary changes for this scope." I find that this is incorrect: only changes for the current scope *in the currently displayed matrix* are saved. Other changes (made on other pages, and not relevant to the current page) are not saved.

Browser: Firefox 39.0 (Mozilla Firefox for Ubuntu - Canonical - 1.0)
uMatrix: v0.9.2.1

Steps to reproduce:

(Do not use host files.)
Go to imgur.com sign-in page: https://imgur.com/signin
Allow ajax.googleapis.com
Allow rlcdn.com (an advertisement network!)
(The dashboard shows the two temporary entries.)
Go to imgur.com main page: http://imgur.com/
Click on the lock icon ("save all changes ...")

Result: only the ajax.googleapis.com entry is saved. The rlcdn.com remains temporary.
Expected result: according to the tooltip, the rlcdn.com entry should also be saved. But maybe the tooltip is wrong?

Note: rlcdn.com is unfortunately not always shown on imgur.com sign-in page. I first found this issue on my bank web site (it has a framed third-party map to find local branches). I can try to provide other examples if needed.

**Comment by gorhill** [comment]
> But maybe the tooltip is wrong?

Yes, I should change it to something more accurate, like "Save all temporary changes in the current matrix" (or whatever better suggestion anybody proposes).

**Comment by adrienbeau** [comment]
I think it is both concise and accurate. I suppose the tooltip for the eraser icon ("revert temporary changes ...") needs updating too? I haven't tested it.
